From 3c28bfd8fafa721eef0db4b4c78eb0bb970e13bd Mon Sep 17 00:00:00 2001
From: sheng wei <w.sheng@intel.com>
Date: Fri, 1 Feb 2019 10:51:55 +0800
Subject: [PATCH] Allow vendor_inint read plat_file_contexts

vendor_init need to assign lables for files under
/data/vendor

Tracked-On:

Signed-off-by: ji, zhenlong z <zhenlong.z.ji@intel.com>
---
 sepolicy/vendor_init.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sepolicy/vendor_init.te b/sepolicy/vendor_init.te
index aee5f7b..4f903bf 100644
--- a/sepolicy/vendor_init.te
+++ b/sepolicy/vendor_init.te
@@ -1,3 +1,4 @@
 allow vendor_init self:global_capability_class_set sys_module;
 allow vendor_init vendor_file:system module_load;
 allow vendor_init kernel:key search;
+allow vendor_init file_contexts_file:file map;
-- 
2.20.1

